codespell_lib/data/dictionary.txt

```diff
@@ -26434,6 +26488,8 @@ unrelying->underlying
 unrepentent->unrepentant
 unrepetant->unrepentant
 unrepetent->unrepentant
+unreplacable->non-replaceable
```

Comment:
This appears to be a real word: https://www.lexico.com/definition/unreplaceable

Irreplaceable would be another suggestion too anyway.